price_transparency/hospitals/machine_readable_links.csv

```diff
@@ -1,184 +1,365 @@
+Penn Presbyterian Medical Center,Penn Presbyterian Medical Center, hospital, https://www.pennmedicine.org/-/media/documents%20and%20audio/non%20patient%20instructions/health%20system/penn%20presbyterian%20medical%20center_23_2810852_march2021a.ashx?la=en,up,,,,csv,,,PA,, 2022-04-14,
+Children's Hospital of Philadelphia,Children's Hospital of Philadelphia, hospital, https://media.chop.edu/data/files/finance/23-1352166_Children%E2%80%99sHospitalofPhiladelphia_standardcharges.csv,up,https://www.chop.edu/centers-programs/billing-and-insurance/understanding-hospital-charges,,,csv,,,PA,, 2022-04-14,
+Pennsylvania Hospital,Pennsylvania Hospital, hospital, https://www.pennmedicine.org/-/media/documents%20and%20audio/non%20patient%20instructions/health%20system/pahosp%20price%20list.ashx?la=en,up,,,,csv,,,PA,, 2022-04-14,
+Penn Medicine Chester County Hospital,Penn Medicine Chester Counter Hospital, hospital, https://www.pennmedicine.org/-/media/documents%20and%20audio/non%20patient%20instructions/health%20system/chester%20county%20hospital_23_0469150_march2021a.ashx?la=en,up,,,,csv,,,PA,, 2022-04-14,
+Penn Medicine Lancaster General Health,Penn Medicine Lancaster General Health, hospital, https://www.pennmedicine.org/-/media/documents%20and%20audio/non%20patient%20instructions/health%20system/lancaster%20general%20hospital_23_1365353_march2021a.ashx?la=en,up,,,,csv,,,PA,, 2022-04-14,
+UCLA Santa Monica Medical Center,UCLA Santa Monica Medical Center, hospital, https://uclahealth.s3-us-west-1.amazonaws.com/price-transparency/954941687_SantaMonicaUCLAMedicalCenter_StandardChargesjson.zip,up,,,,zip/json,,,CA,, 2022-04-14,
+UCLA Ronald Reagan Medical Center,UCLA Ronald Reagan Medical Center, hospital, https://uclahealth.s3-us-west-1.amazonaws.com/price-transparency/956006143_RonaldReaganUCLAMedicalCenter_StandardChargesjson.zip,up,,,,zip/json,,,CA,, 2022-04-14,
+UCLA Resnick Neuropsychiatric Hospital,UCLA Resnick Neuropsychiatric Hospital, hospital, https://uclahealth.s3-us-west-1.amazonaws.com/price-transparency/956006143_ResnickNeuropsychiatricHospital_StandardChargesjson.zip,up,,,,zip/json,,,CA,, 2022-04-14,
+,Chestnut Hill Hospital,hospital,https://chestnuthill.towerhealth.org/app/files/public/6a309db7-fe61-4180-8247-c30f6c536d8a/82-2108660_ChestnutHillHospital_StandardCharges2022.csv,up,https://towerhealth.org/locations/chestnut-hill-hospital/billing/hospital-pricing-chestnut-hill-hospital,,82-2108660_ChestnutHillHospital_StandardCharges2022.csv,csv,,,PA,,2022-04-20,
+,Albert Einstein Medical Center, hospital,https://www.einstein.edu/upload/docs/231396794-1902805245_albert-einstein-medical-center_standardcharges.xlsx,up,https://www.einstein.edu/?id=2141&sid=1,,231396794-1902805245_albert-einstein-medical-center_standardcharges.xlsx,xlsx,FALSE,"File type is xlsx. Proprietary Excel format, and not accepted afaik",PA,,2022-05-02,

```

Comment:
It looks like all occurrences of `false`in the `meets_standard` column were changed to `FALSE` (and new rows were added using this format). I think this is inconsistent with the existing schema. Can you update entries in this column to read `true`, `false`, or be blank?